tests/models/qwen2/colqwen2/test_colqwen2_e2e.py

```diff
@@ -0,0 +1,53 @@
+from typing import cast
+
+import pytest
+import torch
+from PIL import Image
+
+from colpali_engine.models import ColQwen2, ColQwen2Processor
+from colpali_engine.utils.torch_utils import get_torch_device
+
+
+@pytest.fixture(scope="module")
+def model_name() -> str:
+    return "vidore/colqwen2-base"
```

Comment:
@ManuelFay Actually just noticed this model is the non-trained version of ColQwen2. We can keep this now, FYI I'll change it to your new checkpoint when you'll be happy with it!